cscs-checks/libraries/io/netcdf_compile_run.py

```diff
@@ -66,11 +66,8 @@ def setup(self, partition, environ, **job_opts):
                 self.build_system.ldflags = ['-B' + self.linkage]
         else:
             self.build_system.ldflags = ['-%s' % self.linkage]
-            if (environ.name == 'PrgEnv-pgi' and
-                # NOTE: Workaround to fix static linking for C++ with
-                # PrgEnv-pgi
-                self.lang == 'cpp' and
-                self.linkage == 'static'):
-                self.build_system.ldflags += ['-lstdc++']
+            if environ.name == 'PrgEnv-pgi' and self.lang == 'cpp':
+                # NOTE: Workaround to fix compiling for PrgEnv-pgi
```

Comment:
- `NOTE` -> `FIXME`
- `compiling` -> `compilation`

Please put also the the Cray case number in the comment.